README.md

```diff
@@ -20,28 +20,28 @@ By default, firectl searches `PATH` for the firecracker binary. The location of
 
 ```
 Usage:
-  firectl
+  firectl [OPTIONS]
 
 Application Options:
       --firecracker-binary=     Path to firecracker binary
-      --kernel=                 Path to the kernel image (default: ./vmlinux)
-      --kernel-opts=            Kernel commandline (default: ro console=ttyS0 noapic reboot=k panic=1 pci=off nomodules)
+      --kernel=                 Path to the kernel image
```

Comment:
Any particular reason for removing the defaults?

Reply:
Nope, apparently the help text you get from doing firectl with no args and firectl -h are not the same, the no args help text doesn't list the defaults.